nexus/tests/integration_tests/role_assignments.rs

```diff
@@ -9,50 +9,92 @@ use http::StatusCode;
 use nexus_test_utils::http_testing::AuthnMode;
 use nexus_test_utils::http_testing::NexusRequest;
 use nexus_test_utils::resource_helpers::create_organization;
+use nexus_test_utils::resource_helpers::create_project;
 use nexus_test_utils::ControlPlaneTestContext;
 use nexus_test_utils_macros::nexus_test;
 use omicron_common::api::external::ObjectIdentity;
 use omicron_nexus::authn::USER_TEST_UNPRIVILEGED;
-use omicron_nexus::authz::OrganizationRoles;
+use omicron_nexus::authz;
 use omicron_nexus::external_api::shared;
 use omicron_nexus::external_api::views;
 
-// TODO-coverage A more comprehensive test would be useful when we have proper
-// Silo users
 #[nexus_test]
-async fn test_role_assignments_basic(cptestctx: &ControlPlaneTestContext) {
+async fn test_role_assignments_organization(
+    cptestctx: &ControlPlaneTestContext,
+) {
     let client = &cptestctx.external_client;
     let org_name = "test-org";
     create_organization(client, org_name).await;
     let org_url = format!("/organizations/{}", org_name);
+    do_test::<_, views::Organization>(
```

Comment:
I think I'd prefer an explicit type for `T` here (and below on line 47) rather than a wildcard. Not a big deal, just might save some brain cycles when reading.

Reply:
Will do!